float32[] object_pose
float32[] destination_pose
bool is_vertical
bool tip_pick
---
bool success